src/jimgw/samplers/blackjax/ns_aw.py

```
@@ -0,0 +1,222 @@
+            raise ValueError(
+                "log_prior_fn must return 0.0 for all points in [0, 1]^n_dims. "
+            )
+        if not jnp.all(jnp.isnan(lp_out) | jnp.isneginf(lp_out)):
+            raise ValueError(
+                "log_prior_fn must return -inf for all points outside [0, 1]^n_dims. "
+            )
+
+    def sample(
+        self,
+        rng_key: Key,
+        initial_position: Float[Array, "n_live n_dims"],
```

Ruff (0.15.12)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)